addons/html_editor/static/lib/DOMpurify.js

```diff
@@ -0,0 +1,2330 @@
+/*! @license DOMPurify 3.0.9 | (c) Cure53 and other contributors | Released under the Apache license 2.0 and Mozilla Public License 2.0 | github.com/cure53/DOMPurify/blob/3.0.9/LICENSE */
```

Comment:
You probably need to apply a patch to this lib.
See https://github.com/odoo/odoo/commit/e82e96f17367829545ddbf93793f75e89397186d